doc/nrf/releases_and_maturity/migration/migration_guide_2.9.0-nRF54H20-1.rst

```diff
@@ -127,41 +88,77 @@ nrfutil
 
-  * ``nrfutil device`` has been updated to version 2.7.10.
+  * nRF Util ``device`` command has been updated to v2.7.10.
 
-    Install the nRF Util ``device`` command version 2.7.10 as follows::
+    Install the nRF Util ``device`` command v2.7.10 as follows::
 
        nrfutil install device=2.7.10 --force
 
     For more information, consult the `nRF Util`_ documentation.
 
-nrfutil trace
-+++++++++++++
+nRF Util trace
+++++++++++++++
 
 .. toggle::
 
-  * ``nrfutil trace`` has been updated to version 3.10.0.
+  * nRF Util ``trace`` command has been updated to v3.1.0.
 
-    Install the nRF Util ``trace`` command version 3.10.0 as follows::
+    Install the nRF Util ``trace`` command v3.1.0 as follows::
 
-       nrfutil install trace=3.10.0 --force
+       nrfutil install trace=3.1.0 --force
 
     For more information, consult the `nRF Util`_ documentation.
 
-nrfutil suit
-++++++++++++
+nRF Util suit
++++++++++++++
 
 .. toggle::
 
-  * ``nrfutil suit`` has been updated to version 0.9.0.
+  * nRF Util ``suit`` command has been updated to v0.9.0.
 
-    Install the nRF Util ``suit`` command version 0.9.0 as follows::
+    Install the nRF Util ``suit`` command v0.9.0 as follows::
 
        nrfutil install suit=0.9.0 --force
 
     For more information, consult the `nRF Util`_ documentation.
+
+nRF54H20 SoC binaries
++++++++++++++++++++++
+
+.. toggle::
+
+  * The *nRF54H20 SoC binaries* bundle has been updated to version 0.8.0.
+
+    .. note::
+       The nRF54H20 SoC binaries only support specific versions of the |NCS| and do not support rollbacks to a previous version.
+       Upgrading the nRF54H20 SoC binaries on your development kit might break the DK's compatibility with applications developed for previous versions of the |NCS|.
+       For more information, see :ref: `abi_compatibility`.
+
+    To update the SoC binaries bundle of your development kit while in Root of Trust, do the following:
+
+    1. Download the `nRF54H20 SoC binaries v0.9.0`_.
+
+       .. note::
+          On macOS, ensure that the ZIP file is not unpacked automatically upon download.
+
+    #. Purge the device as follows::
+
+          nrfutil device recover --core Application --serial-number <serial_number>
+          nrfutil device recover --core Network --serial-number <serial_number>
+          nrfutil device reset --reset-kind RESET_PIN --serial-number <serial_number>
+
+    #. Run ``west update``.
+    #. Move the correct :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+
+          nrfutil device x-suit-dfu --firmware nrf54h20_soc_binaries_v0.9.0.zip --serial-number <serial_number>
+
+    #. Purge the device again as follows::
+
+          nrfutil device recover --core Application --serial-number <serial_number>
+          nrfutil device recover --core Network --serial-number <serial_number>
+          nrfutil device reset --reset-kind RESET_PIN --serial-number <serial_number>
```

Comment:
The `nrf-regtool` requirements in the NCS build system are incorrectly set for SoC binaries v0.9.0. The STM logging functionality does not work with the minimum required version of the `nrf-regtool` and that is `v8.0.0`. Should we mention in the migration guide that the user should migrate to the newest version of `nrf-regtool` (`v8.1.2`)?

Reply:
I would also recommend bumping the minimum required version in the build system so users of this NCS release will get a clear build failure if they attempt to build nRF54H projects with an incorrect `nrf-regtool` version:

https://github.com/nrfconnect/sdk-zephyr/blob/main/modules/hal_nordic/CMakeLists.txt#L18

Reply:
See the similar migration note for the nrf-regtool from the NCS 2.8 migration guide:

https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst#nrf-regtool

Reply:
@kapi-no It was explicitely requested by @hakonfam  (and others from the tools team) to avoid any mention of hard regtool versions. Updating the toolchain will update the requirements. To my knowledge, as long as the version mentioned in requirements.txt is correctly set, the current steps in the migration guide will be correct. However, I think requirements.txt should not be updated in this PR, if possible.